run-test.sh

```diff
@@ -214,6 +172,8 @@ runtest()
   then
     rm mscorlib.ni.dll
   fi
+
+  chmod +x ./corerun
```

Comment:
I was only seeing this in CI.  My guess is that when we copy the CoreCLR build (which contains corerun) from the Jenkins server, the execute bit is not preserved.

I think that long term this will also be needed, since when you restore corerun from a nuget package, it will not be executable.